DQMOffline/Trigger/python/ExoticaMonitoring_Client_cff.py

```diff
@@ -16,6 +16,26 @@
   
 )
 
+
+muonEfficiency = DQMEDHarvester("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/Muon/*"),
```

Comment:
I don't follow this comment Mia, so for other Muon paths, they do the efficiency using tag and probe method, and for these paths I use orthogonal method, so thats why I created new code .